drivers/note/notelog_driver.c

```diff
@@ -320,37 +410,20 @@ void sched_note_spinlock(FAR struct tcb_s *tcb,
 #endif
 
 #ifdef CONFIG_SCHED_INSTRUMENTATION_SYSCALL
-void sched_note_syscall_enter(int nr, int argc, ...)
+static void notelog_syscall_enter(FAR struct note_driver_s *drv, int nr)
 {
-  char buf[128];
-  FAR char *p = buf;
-  va_list ap;
-
-  va_start(ap, argc);
-  while (argc-- > 0)
-    {
-      if (argc)
-        {
-          p += sprintf(p, "%#"PRIxPTR", ", va_arg(ap, uintptr_t));
-        }
-      else
-        {
-          p += sprintf(p, "%#"PRIxPTR, va_arg(ap, uintptr_t));
-        }
-    }
-
-  va_end(ap);
-  syslog(LOG_INFO, "%s@%d ENTER %s\n", g_funcnames[nr], nr, buf);
+  syslog(LOG_INFO, "%s@%d ENTER %s\n", g_funcnames[nr], nr);
 }
 
-void sched_note_syscall_leave(int nr, uintptr_t result)
+static void notelog_syscall_leave(FAR struct note_driver_s *drv, int nr)
```

Comment:
why remove result

Reply:
syslog is a system call, sched_note_syscall_xxx cannot be used in notelog